components/services.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { FaUtensils, FaPlane, FaBus, FaHotel } from 'react-icons/fa';
+
+const Services = () => {
+  return (
+    <section className="">
+      <div className="flex flex-col text-white justify-center items-center">
+        <h2 className="text-center mb-3 text-4xl md:text-5xl font-abhaya">The Perfect Travel</h2>
+        <hr className="w-[180px] md:w-[720px] h-px mx-auto mt-2 md:mb-2 bg-gray-500 border-0 rounded dark:bg-gray-700" />
+        <p className="mx-6 my-4 text-xl font-aloe md:text-lg">
+          We cover all expenses from picking the perfect hotel to flights and destinations.
+        </p>
+      </div>
+      <div className="bg-white grid gap-4 md:grid-cols-2 py-10 md:px-20 px-8 md:gap-x-6 md:gap-y-8">
+        <div className="shadow-lg">
+          <div className="flex justify-evenly text-white py-2 bg-[#1B1D33]">
+            <p className="font-abril">Tour Guide</p>
+            <FaPlane className="text-2xl" />
+          </div>
+          <p className="px-8 py-4 text-black">
+            Lorem ipsum dolor sit amet, consectetur adipisicing elit. Corporis magnam ut ea error perspiciatis magni velit
+            unde tempore quasi? Dignissimos?
+          </p>
+        </div>
+        <div className="shadow-lg">
+          <div className="flex justify-evenly text-white py-2 bg-[#1B1D33]">
+            <p className="font-abril">Hotel</p>
+            <FaHotel className="text-2xl" />
+          </div>
+          <p className="px-8 py-4 text-black">
+            Lorem ipsum dolor sit amet, consectetur adipisicing elit. Corporis magnam ut ea error perspiciatis magni velit
+            unde tempore quasi? Dignissimos?
+          </p>
+        </div>
+        <div className="shadow-lg">
+          <div className="flex justify-evenly text-white py-2 bg-[#1B1D33]">
+            <p className="font-abril">Transport</p>
+            <FaBus className="text-2xl" />
+          </div>
+          <p className="px-8 py-4 text-black">
+            Lorem ipsum dolor sit amet, consectetur adipisicing elit. Corporis magnam ut ea error perspiciatis magni velit
+            unde tempore quasi? Dignissimos?
+          </p>
+        </div>
+        <div className="shadow-lg">
+          <div className="flex justify-evenly text-white font-abril py-2 bg-[#1B1D33]">
+            <p className="font-abril">Meal</p>
+            <FaUtensils className="text-2xl" />
+          </div>
+          <p className="md:px-8 py-4 text-black">
+            Lorem ipsum dolor sit amet, consectetur adipisicing elit. Corporis magnam ut ea error perspiciatis magni velit
+            unde tempore quasi? Dignissimos?
+          </p>
+        </div>
+      </div>
```

Comment:
@NurkaAmre Excellent job implementing the user services section 👍 However, instead of nested `div` you can you can use an unordered list and make sure you follow the Figma design template to design the UI.

Reply:
@raihan2bd I made changes for for unoreder list on figma design icons are images not react icons and its was hard too see color if blue is on dark blue that's why i leave with white